implementations/go/handlers/baseHandler.go

```diff
@@ -0,0 +1,11 @@
+package handlers
+
+import (
+	"fmt"
+	"net/http"
+)
+
+//HomeHandler - function to handle lading page routes
+func HomeHandler(w http.ResponseWriter, r *http.Request) {
+	fmt.Fprint(w, "welcome to the movie guide bot challenge")
```

Comment:
Hmmmmm... since most people have chosen Soccergist i think we shouldnt break their heart. I just used that because i did not know i will be contributing here eventually. I just set that up to follow my own guide, but we can use that later on once this is completed... I have just always been a fan of movie related stuffs. Moreso, i dont want to be googling info about my favorite series everytime. Thats d point of the movie guide.

Reply:
I agree, total 💯

Reply:
Coolio 👍